packages/react/src/components/Accordion/AccordionTitle.tsx

```diff
@@ -24,9 +29,18 @@ export interface AccordionTitleProps
   /** Whether or not the title is in the open state. */
   active?: boolean
 
+  /** If at least one panel needs to stay active and this title does not correspond to the last active one. */
+  cannotBeClosed?: boolean
```

Comment:
to be consistent with expand/collapse terminology, maybe just `cannotBeCollapsed`

Reply:
added `canBeCollapsed` and inverted the logic